inf-ruby.el

```diff
@@ -282,6 +284,20 @@ The following commands are available:
       (inf-ruby-remove-in-string (buffer-substring (point) end)
                                  inf-ruby-prompt-pattern))))
 
+(defun inf-ruby-buffer ()
+  "Return inf-ruby buffer for the current buffer or project."
+  (catch 'buffer
+    (let ((current-dir         (file-truename default-directory))
```

Comment:
I'm just double checking that I don't compare for example "~/projects/foo" with "/home/asok/projects/foo".